action-core/build.gradle

```diff
@@ -39,8 +39,9 @@ dependencies {
     api project(':await')
     api project(':qr-code')
     api project(':redirect')
-    compileOnly project(':wechatpay')
+    api project(':twint')
```

Comment:
Should we make this `compileOnly`? It's only going to be used in some markets and it's not even the default Twint flow (looking at the `shouldUseSdk` configuration)

Reply:
Good point 👍 `shouldUseSdk` is true by default though, so it would be the default flow. If `:twint` is not included by the merchant and they forget to set `shouldUseSdk` to false, then the whole flow wouldn't work as it can't fall back to web in this case 🤔

Reply:
> shouldUseSdk is true by default though, so it would be the default flow

Good point, I missed that.

> the whole flow wouldn't work as it can't fall back to web in this case

I think it would be a wrong integration from the merchant. So it's better to fail rather than use the SDK and risk them getting unwanted results. Added to the fact that we avoid shipping this `aar` to everyone using components 🙃

Reply:
Alright, I'll change it to `compileOnly`